lib/msf/core/post/file.rb

```diff
@@ -506,14 +506,15 @@ def rm_rf(*remote_dirs)
   # @param old_file [String] Remote file name to move
   # @param new_file [String] The new name for the remote file
   def rename_file(old_file, new_file)
+    raise "File to be renamed doesn't exists" unless exist?(old_file)
```

Comment:
Since we're returning a boolean value to indicate whether or not the operation was successful, maybe this should actually just log a message and return `false` instead of raising an exception.

Reply:
okay cool :+1: